opencensus-proto/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "opencensus-proto"
 version = "0.1.0"
 authors = ["The OpenCensus Authors"]
+license = "Apache-2.0"
```

Comment:
do we know what license the opencensus protobuf API definition is distributed under? does this need to match that?

Reply:
It matches https://github.com/census-instrumentation/opencensus-proto